src/clr-angular/emphasis/alert/_alert.clarity.scss

```diff
@@ -355,7 +355,7 @@
 
     .alert-app-level {
       .alert-actions {
-        margin-left: 0;
+        margin-left: 45px;
```

Comment:
@mathisscott - should this be the equivalent REM unit? e.g `1.875rem` for ~45px?

Reply:
Ah, yes it should. Good catch.

Reply:
Please update this value to rems...

Reply:
Done. But for some reason, I am getting dco-required event when I am adding "Name and e-mail id" as the last line in the commit. 

Am I missing something?